00_hello_world/main_test.go

```diff
@@ -18,7 +18,15 @@ func TestMainOutput(t *testing.T) {
 	main()
 
 	// Then
-	expected := strconv.Quote("Hallo du schöne Welt!\n")
```

Comment:
Also test `fib`, not just `main` (unit tests).
Test corner cases - i: -1, -0, 1, very large number